examples/deferred_processing/automated_deferred_processing/branch-mover.json

```diff
@@ -0,0 +1,33 @@
+{
+  "pipeline": {
+    "name": "branch-mover"
+  },
+  "input": {
+    "cron": {
+        "name": "tick",
+        "spec": "@every 1m",
+        "overwrite": true
+    }
+  },
+  "transform": {
+      "cmd": ["sh" ],
+      "stdin": [
+          "echo '{\"pachd_address\": \"grpc://pachd:650\"}' | pachctl config set context default --overwrite",
+          "echo ${PACHYDERM_AUTH_TOKEN} | pachctl auth use-auth-token",
+          "pachctl create branch edges_dp@master --head dev"
+      ],
+      "image": "pachyderm/pachctl:1.11.0",
+      "secrets": [ {
+          "name": "pachyderm-user-secret",
+          "mount_path": "/pachyderm-user-secret"
+      },
```

Comment:
is this first secret necessary? I don't see it used anywhere (versus the env var below.)

Reply:
This is my ignorance. I guess I can just use it as an env var, and not worry about the mount!